Automattic/woocommerce-payments
# Spike: Update WooPay order on order success page

Spike. Main PR in 1223-gh-Automattic/woopay

## Changes proposed in this Pull Request









- Process WooPay order when customer arrives on the order success page.

## Testing instructions





**Make sure you've checked out `add/no-update-request-to-merchant-store` in WooPay and `trunk` on the server.**

1. Make sure checkout via WCPay works as intended.
2. Make sure checkout via WooPay:
    1. Works as intended.
    2. Updates the order with correct information (WooPay order, last 4 digits of CC, order notes)
    3. Order received page shows WooPay info and last 4 digits. 

-------------------


**Post merge**
